changelog/fragments/1696955150-Slow-down-agent-monitoring-metrics-interval-to-60s.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
```

Comment:
I think, _technically_ this should be an `enhancement` as it's not an entirely new `feature`?

Reply:
I didn't check the full list of `kind` values we could specify, was mostly thinking of "it's not a bug" xD
Will change it.

Reply:
fixed in 80135324d204c2cbb21e44cfe329145ab2f64e90